.github/workflows/security.yml

```diff
@@ -0,0 +1,94 @@
+name: Scan container images
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+  schedule:
+    - cron: '0 4 * * 1'  # Every Monday at 9:30 IST
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: paralusio/paralus
+
+jobs:
+
+  build-scan-paralus:
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+    runs-on: ${{ matrix.os }}
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: 'go.mod'
+
+      - name: Build an image from Dockerfile
+        run: |
+          docker build -f "Dockerfile" -t ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ github.sha }} .
```

Comment:
We can just tag image as `paralus:${{ github.sha }}` as we are not pushing image to registry. With this, we no longer need to set `REGISTRY` and `IMAGE_NAME` env variables above - lesser lines.

Reply:
renamed the tag